shoop/admin/modules/products/__init__.py

```diff
@@ -87,14 +87,14 @@ def get_search_results(self, request, query):
             url = reverse("shoop_admin:product.new")
             if " " in query:
                 yield SearchResult(
-                    text="Create product called '%s'" % query,
+                    text="New product called '%s'" % query,
```

Comment:
Other buttons have the entity in title case, also using double quotes seems more appropriate than single quotes and this probably should be translated, i.e. the text here should be `_('New Product called "%s"')`.